### PR TITLE
Fix broken links (part 3 of 3)

### PR DESCRIPTION
**Pull request by gewarren**
Fix broken links according to OPS report.

Deletes web service binding tutorial that points to non-existent web service and has low page views (64 last 30 days).

Contributes to #17649.

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
These all LGTM @gewarren 

You can :shipit: when ready.

File: docs/machine-learning/how-to-guides/use-gams-for-model-explainability.md [hunk 1]
```diff
@@ -45,4 +45,4 @@ for (int i = 0; i < myFeatureBins.Length; i++)
 
 ![Generalized Additive Models shape function graph](./media/use-gams-for-model-explainability/gam-shape-function-graph.png)
 
-For a sample of how to train a GAM model and inspect and interpret the results, see [the dotnet/machinelearning GitHub repository](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/GeneralizedAdditiveModels.cs).
+For an example of how to train a GAM model and inspect and interpret the results, see the [binary classification trainer sample](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/Gam.cs).
```
Comment (gewarren) [inline docs/machine-learning/how-to-guides/use-gams-for-model-explainability.md:48]: Before I merge this PR, @natke is this sample appropriate here?

Comment (gewarren) [reply docs/machine-learning/how-to-guides/use-gams-for-model-explainability.md:48]: @luisquintanilla Does this sample link look appropriate?

Comment (natke) [reply docs/machine-learning/how-to-guides/use-gams-for-model-explainability.md:48]: Yes that looks right @gewarren

Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/use-gams-for-model-explainability.md:48]: Thanks for the reply @natke. Yes @gewarren that sample is appropriate

**Pull request merged**